var/spack/repos/builtin/packages/py-meldmd/package.py

```diff
@@ -1,28 +1,88 @@
-# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import os
+
 from spack.package import *
 
 
-class PyMeldmd(PythonPackage, CudaPackage):
+class PyMeldmd(CMakePackage, PythonPackage, CudaPackage):
     """MELD is a tool for inferring the structure of
     biomolecules from sparse, ambiguous, or noisy data."""
 
     homepage = "http://meldmd.org/"
     url = "https://github.com/maccallumlab/meld/archive/refs/tags/0.4.20.tar.gz"
 
+    version("0.6.1", sha256="aae8e5bfbdacc1e6de61768a3298314c51575cda477a511e98dc11f5730fd918")
     version("0.4.20", sha256="8c8d2b713f8dc0ecc137d19945b3957e12063c8dda569696e47c8820eeac6c92")
 
+    extends("python")
+
     depends_on("python@3.6:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
     depends_on("amber")
-    depends_on("openmm")
+    depends_on("openmm+cuda")
     depends_on("py-netcdf4", type=("build", "run"))
-    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-numpy@:1.23", type=("build", "run"))
     depends_on("py-scipy", type=("build", "run"))
     depends_on("py-scikit-learn", type=("build", "run"))
     depends_on("py-parmed", type=("build", "run"))
     depends_on("py-tqdm", type=("build", "run"))
     depends_on("py-mpi4py", type=("build", "run"))
+    depends_on("py-pip", type="build")
+
+    # C++ / CUDA
+    depends_on("eigen", type=("build", "run"))
```

Comment:
Eigen has no `prefix.bin` directory, so there's nothing to add to the `PATH`. Shouldn't this be:
```suggestion
    depends_on("eigen", type="link")
```

Reply:
I will test that change